fleet/agent_service.go

```diff
@@ -149,11 +160,11 @@ func (svc fleetService) RemoveAgent(ctx context.Context, token, thingID string)
 	}
 
 	if errT := svc.mfsdk.DeleteThing(res.MFThingID, token); errT != nil {
-		err = errors.Wrap(err, errT)
+		svc.logger.Error("failed to delete thing", zap.Error(errT))
```

Comment:
Sorry forgot to mention this but please include res.MFThingID as a logger variable, like this:
```
svc.logger.Error("failed to delete thing", zap.Error(errT), zap.String('thing_id', res.MFThingID))
```